packages/frontend/src/routes/SetupRecoveryImplicitAccountWrapper.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import SetupRecoveryImplicitAccount from '../components/accounts/create/implicit_account/SetupRecoveryImplicitAccount';
+import { redirectTo } from '../redux/actions/account';
+
+export function SetupRecoveryImplicitAccountWrapper() {
+    const dispatch = useDispatch();
+    return (
+        <SetupRecoveryImplicitAccount
```

Comment:
I think we'd like to land this before we have tor.us support implemented, right?  If so, don't we need a case for `email` here?